demos/setup-project.sh

```diff
@@ -22,8 +22,9 @@ PROJECT=sample
 # Self-hosted (open source) installations require project creation.
 
 # Delete and re-create the project to get a fresh start.
-registry rpc admin delete-project --name projects/$PROJECT --force
+registry rpc admin delete-project --name projects/$PROJECT --force >& /dev/null
```

Comment:
could we use  `set -e` at the beginning of the script to allow the script to continue on error.

And use
registry rpc admin delete-project --name projects/$PROJECT --force || true

Reply:
Alternatively, are we avoiding `registry delete` for some reason? That might be an easier option.

Reply:
That won't suppress the error message that's printed by the `delete-project` handler -- I think we would have to modify the generated code to do that (thus the redirect instead).

Reply:
This also alerts us that the generated handler doesn't return an error for grpc errors. It seems that it succeeds because the grpc call succeeds, regardless of response code.

Reply:
@seaneganx It's not really necessary to use `registry delete` and since there's no corresponding `registry create`, I think the symmetrical commands are easier to follow.

Reply:
@seaneganx Also, `registry delete` currently does not delete projects.
```
% registry delete projects/sample
 FATAL[0000] Failed to match or handle command error=unsupported resource name: see the 'apg registry delete-' subcommands for alternatives uid=31e7858b
```